### PR TITLE
Cl Conf: Salt bundle admonitions cleanup

### PR DESCRIPTION
**Pull request by 0rnela**
# Description

Part of larger cleanup.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/client-configuration/pages/contact-methods-saltbundle.adoc [hunk 1]
```diff
@@ -36,8 +36,9 @@ For more information, see xref:client-configuration:registration-cli.adoc#_salt_
 
 [IMPORTANT]
 ====
-To bootstrap a client with the Salt Minion managed by the Salt Master other than {productname} Server it is recommended to use [command]``mgr-bootstrap`` [option]``--force-bundle`` when generating the bootstrap script, or to set [option]``FORCE_VENV_SALT_MINION`` to `1` in the bootstrap script.
-For bootstrapping with {webui} [literal]``mgr_force_venv_salt_minion`` set to [literal]``true`` pillar can be specified globally.
+* To bootstrap a client with the Salt Minion managed by the Salt Master other than {productname} Server it is recommended to use [command]``mgr-bootstrap`` [option]``--force-bundle`` when generating the bootstrap script, or to set [option]``FORCE_VENV_SALT_MINION`` to `1` in the bootstrap script.
+* For bootstrapping with {webui} [literal]``mgr_force_venv_salt_minion`` set to [literal]``true`` pillar can be specified globally.
+
 For more information, see xref:specialized-guides:salt/salt-states-and-pillars.adoc[Salt States and Pillars].
```
Comment (keichwa) [inline modules/client-configuration/pages/contact-methods-saltbundle.adoc:42]:
```suggestion
For more information, see xref:specialized-guides:salt/salt-states-and-pillars.adoc[].
```

File: modules/client-configuration/pages/contact-methods-saltbundle.adoc [hunk 2]
```diff
@@ -85,13 +86,15 @@ The Python 3 deployed with the Bundle is used to handle Salt SSH session on the
 [IMPORTANT]
 ====
 The bootstrap repository must be created before bootstrapping the client with {webui}. Salt SSH is using the Salt Bundle taken from the bootstrap repository based on the detected target operating system.
+
 For more information, see xref:client-configuration:bootstrap-repository.adoc#_prepare_to_create_a_bootstrap_repository[Prepare to Create a Bootstrap Repository]
```
Comment (keichwa) [inline modules/client-configuration/pages/contact-methods-saltbundle.adoc:90]:
```suggestion
The bootstrap repository must be created before bootstrapping the client with {webui}.
Salt SSH is using the Salt Bundle taken from the bootstrap repository based on the detected target operating system.

For more information, see xref:client-configuration:bootstrap-repository.adoc#_prepare_to_create_a_bootstrap_repository[].
```

File: modules/client-configuration/pages/contact-methods-saltbundle.adoc [hunk 3]
```diff
@@ -111,7 +114,9 @@ By default, [command]``salt <minion_id> pip.install <package-name>`` installs th
 [NOTE]
```
Comment (keichwa) [inline modules/client-configuration/pages/contact-methods-saltbundle.adoc:114]: I'd convert this NOTE to a proper section or one or two paragraphs

**Review by keichwa — CHANGES_REQUESTED** [review]

**Review by jcayouette — CHANGES_REQUESTED** [review]
I would adjust as karl proposes.

**Pull request closed without merging**